### ISSUE TITLE
Add trader joe links / dropdown on mobile

### ISSUE THREAD

**Issue by IljaDaderko** [Wonderland-Money/wonderland-frontend#11]
**Is your feature request related to a problem? Please describe.**
Dropdown that links to trader joe swap for TIME / MEMO tokens is missing on mobile.

**Describe the solution you'd like. Include clear acceptance criteria.**
Perhaps add it somewhere in that main hamburger menu?

**Is any UX Design input needed?**
-

**Describe alternatives you've considered**
-

**Tech Notes/Testing Notes**
-

**Out of scope**
-

**Additional context**
-

**Comment by nebuk4d** [comment]
Hey @IljaDaderko 🙋🏻‍♂️ , the link should be there if you click on the time button right next to the hamburger menu.

**Comment by IljaDaderko** [comment]
Hey, yep it is on desktop, this issue is solely for mobile, as that time button doesn't appear. The only difference I see in our screenshots is that I am connected and you are not, perhaps that could be the issue?

This is on iOS / Metamask browser
![IMG_0083](https://user-images.githubusercontent.com/3154053/141264222-4cbf4bd0-54d9-49e7-879a-f552deb3bd10.jpg)

**Comment by nebuk4d** [comment]
hm yeah weird. I also tested now on iOS and metamask browser and still it is there 🤔 Are u using iPad or iPhone?
![IMG_1123](https://user-images.githubusercontent.com/52011202/141267784-277d3e3f-a064-4734-83f8-50ccc22d4eb3.PNG)

**Comment by nebuk4d** [comment]
The button is not printed on smaller screens. (width 400px)
That could be the reason.

See: https://github.com/Wonderland-Money/wonderland-frontend/blob/master/src/components/Header/index.tsx#L55

**Comment by IljaDaderko** [comment]
That's probably it, do you reckon it should be printed when users are in a "connected" state perhaps? As screen real estate allows for it and I guess it makes sense to want to add the tokens once you connected

**Comment by nebuk4d** [comment]
Yeah I think it makes sense. I see that beginning with a width of 388px the buttons start to not fit properly. So till that value it might can be tweaked a bit otherwise the buttons need to be changed or we live with the newLine which I personalld don't like much.
What iPhone do you have or better what screen size?

**Comment by IljaDaderko** [comment]
Yo, sorry for late reply, I'm on iPhone 12 Pro, I believe its 375px width. Which would still not fit in this case. What about making that button an icon instead of "TIME" text? Like icon of a contract or something that would make sense for tokens and trader joe?

Also, if it is at least shown in "connected" state, that would help a lot. I understand that in disconnected state "Connect Wallet" button takes up a lot of space.

**Comment by nebuk4d** [comment]
https://github.com/Wonderland-Money/wonderland-frontend/pull/14

Maybe just decreasing the padding a bit, so that the range of devices supported grows, is more efficient right now, without changing to much of the design.
